src/backend/TrafficCourts/Citizen.Service/Services/Tickets/Search/Rsi/IRoadSafetyTicketSearchApi.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Refit;
+using TrafficCourts.Citizen.Service.Services.Tickets.Search.Common;
+
+namespace TrafficCourts.Citizen.Service.Services.Tickets.Search.Rsi
+{
+    public interface IRoadSafetyTicketSearchApi
+    {
+        [Get("/paybc/vph/rest/PSSGVPHPAYBC/vph/")]
```

Comment:
not sure if these routes could change, but it was hard coded in the API documentation